packages/dd-trace/test/plugins/externals.json

```diff
@@ -170,6 +170,26 @@
       "name": "react-dom",
       "dep": true
     }
+  ],
+   "passport-http": [
+     {
+       "name": "passport-http",
+       "versions": ["0.3.0"]
+     },
+     {
+       "name": "passport",
+       "versions": [">=0.4.1"]
+     }
+   ],
+   "passport-local": [
+    {
+      "name": "passport-local",
+      "versions": ["1.0.0"]
+    },
+    {
+      "name": "passport",
+      "versions": [">=0.4.1"]
+    }
```

Comment:
are you sure that's how it works ? i personally forgot a bit how that externals file works, maybe ask Roch, it looks a bit weird to me

Reply:
I'm not really familiar with its inner workings but I followed the same patter as in mocha:
```
"mocha": [
    {
      "name": "mocha",
      "versions": [">=5.2.0"]
    },
    {
      "name": "mocha-each",
      "versions": [">=2.0.1"]
    }
],
```

or jest:
 ```
"jest": [
    {
      "name": "jest",
      "versions": [">=24.8.0"]
    },
    {
      "name": "jest-environment-jsdom",
      "versions": [">=24.8.0"]
    },
    {
      "name": "jest-environment-node",
      "versions": [">=24.8.0"]
    },
    {
      "name": "jest-circus",
      "versions": [">=24.8.0"]
    },
    {
      "name": "@jest/globals",
      "versions": [">=26.5.0"]
    }
]
```

Reply:
I'll ask on the guild channel